browser/js/common/factories/snippet-factory.js

```diff
@@ -106,12 +106,45 @@ app.factory("Snippet", function($firebaseObject, AuthService, Users, $rootScope,
             })
             updates[`/users/${snippet.team}/snippets/asManager/${snippetId}`] = null;
             updates[`/users/${snippet.owner}/snippets/asOwner/${snippetId}`] = null;
+            updates[`/users/${snippet.owner}/snippets/asCollaborator/${snippetId}`] = null;
```

Comment:
Another neat es6 trick. You can embed `[]` for an object literal *key* to specify a dynamic value. So this could be ``updates = {[`/users/${snippet.owner}/snippets/asCollaborator/${snippetId}`]: null}``. Not at all necessary, but thought it would be useful info for you all.